### PR TITLE
Broker class and necessary configuration

### PR DESCRIPTION
**Pull request by brngylni**
Regarding to #14 @abompard

### CODE REVIEW

File: webhook_to_fedora_messaging/config.py [hunk 1]
```diff
@@ -4,13 +4,15 @@
 
 from functools import cache
 from pathlib import Path
-
+import os
 from pydantic import BaseModel, ConfigDict, DirectoryPath
 from pydantic_settings import BaseSettings
 
 
 DEFAULT_CONFIG_FILE = _config_file = "/etc/webhook-to-fedora-messaging/webhook-to-fedora-messaging.cfg"
```
Comment (gridhead) [inline webhook_to_fedora_messaging/config.py:12]: I see that you are using the INI-styled configuration here. Could you please include a default configuration with placeholder values for folks to get started with?

Comment (gridhead) [reply webhook_to_fedora_messaging/config.py:12]: Also, @abompard should we stick to an INI-styled configuration or use a more profound TOML-styled configuration? I am fine with either but I wanted to hear your opinion on this.

**Comment by gridhead** [comment]
@brngylni Please refer to https://github.com/fedora-infra/webhook-to-fedora-messaging/pull/2#issuecomment-2134623677 for how commit messages should be written.

**Review by gridhead — COMMENTED** [review]
@abompard, this is not in @brngylni's changes but can we please use context-aware modules for storing configuration instead of global variables?

Take for instance, the section https://github.com/fedora-infra/webhook-to-fedora-messaging/pull/16/files#diff-00c1df60e290e533baec34ac9f6abd4db0fa8b99f04485315dff43023f8049b6R44-R47, could simply be a module-wide variable that can be referenced and changed throughout the project with the use of the following.

In the configuration initialization module named `something.py`,

```python
from webhook_to_fedora_messaging import config

config.some_variable = "SOMETHING"
```

In the configuration using module named `something_else.py`,

```python
from webhook_to_fedora_messaging import config

def do_something() -> str:
    return config.some_variable
```

As long as we do not import the variable like the following cases.

In the configuration initialization module named `something.py`,

```python
from webhook_to_fedora_messaging.config import some_variable

some_variable = "SOMETHING"
```

In the configuration using module named `something_else.py`,

```python
from webhook_to_fedora_messaging.config import some_variable

def do_something() -> str:
    return some_variable
```

We should be able to ensure that the variable is not instantiated with the default value and that the variable retains the value set to it during the runtime.

File: webhook_to_fedora_messaging/config.py (hunk 1 above)
Comment (abompard) [reply webhook_to_fedora_messaging/config.py:12]: We shouldn't use pydantic anyway, because Flask has its own configuration system.

Comment (gridhead) [reply webhook_to_fedora_messaging/config.py:12]: Yeah, I got bamboozled seeing all the Pydantic stuff here and assumed that it was used here.

**Comment by brngylni** [comment]
> @abompard, this is not in @brngylni's changes but can we please use context-aware modules for storing configuration instead of global variables?
> 
> Take for instance, the section https://github.com/fedora-infra/webhook-to-fedora-messaging/pull/16/files#diff-00c1df60e290e533baec34ac9f6abd4db0fa8b99f04485315dff43023f8049b6R44-R47, could simply be a module-wide variable that can be referenced and changed throughout the project with the use of the following.
> 
> In the configuration initialization module named `something.py`,
> 
> ```python
> from webhook_to_fedora_messaging import config
> 
> config.some_variable = "SOMETHING"
> ```
> 
> In the configuration using module named `something_else.py`,
> 
> ```python
> from webhook_to_fedora_messaging import config
> 
> def do_something() -> str:
>     return config.some_variable
> ```
> 
> As long as we do not import the variable like the following cases.
> 
> In the configuration initialization module named `something.py`,
> 
> ```python
> from webhook_to_fedora_messaging.config import some_variable
> 
> some_variable = "SOMETHING"
> ```
> 
> In the configuration using module named `something_else.py`,
> 
> ```python
> from webhook_to_fedora_messaging.config import some_variable
> 
> def do_something() -> str:
>     return some_variable
> ```
> 
> We should be able to ensure that the variable is not instantiated with the default value and that the variable retains the value set to it during the runtime.


I can switch the structure to this if you both agreed on this

**Pull request closed without merging**